lib/identity_config.rb

```diff
@@ -449,6 +450,24 @@ def self.build_store(config_map)
     config.add(:voip_allowed_phones, type: :json)
     config.add(:voip_block, type: :boolean)
 
+    config.add(:irs_attempt_api_auth_tokens, type: :comma_separated_string_list) do |list|
+      salt = SecureRandom.hex(32)
+      cost = config.written_env[:scrypt_cost]
```

Comment:
this line is giving me bad vibes, I am really worried about the portability of secrets referencing each other during initialization like this. How would you feel about just hardcoding our production config cost here, since it's not likely to change anytime soon?

Reply:
The pain point is testing and local development where it'd hash the production value on startup every time. Maybe there's a different place to hold it.

Reply:
Agreed on the bad vibes though